runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContentsClientBridge.java

```diff
@@ -162,6 +163,18 @@ public void setResourceClient(XWalkResourceClientInternal client) {
         mXWalkResourceClient = new XWalkResourceClientInternal(mXWalkView);
     }
 
+    public void setExMediaPlayer(XWalkExMediaPlayerInternal mediaPlayer) {
+        // If it's null, use Crosswalk implementation.
+        if (mediaPlayer != null) {
```

Comment:
If mediaPlayer is not allowed to be null, it should throw an exception here or anything else. It's not intuitive if a null parameter is passed but nothing happened without warning.